cli/http.go

```diff
@@ -108,8 +111,10 @@ func (ac *APIClient) doReq(method, path string, apiVersion int, body io.Reader)
 		req, err = http.NewRequest(method, fmt.Sprintf("%s/%s", ac.APIRoot, path), body)
 	} else if apiVersion == 2 {
 		req, err = http.NewRequest(method, fmt.Sprintf("%s/%s", ac.APIRootV2, path), body)
+	} else if apiVersion == -1 {
```

Comment:
Can this be made available in the API?

Reply:
This route is only used by the UI server at the moment and requires the evergreen settings